packages/core/src/utilities/scroll/hasOverflowStyles.ts

```diff
@@ -1,12 +1,12 @@
 import {getWindow} from '@dnd-kit/utilities';
 
-export function isScrollable(
-  element: HTMLElement,
+export function hasOverflowStyles(
+  element: Element,
```

Comment:
`HTMLElement` is a subset of `Element`, so `Element` should be fine to use here (it won't break the public API).